src/routes/components/FormItemPage.tsx

```diff
@@ -78,19 +79,19 @@ export default function FormItemPage() {
     },
     {
       name: "helpText",
-      type: "string",
+      type: "string | slot",
```

Comment:
The React component would instead accept a `ReactNode`, which will be injected into the appropriate slot. For that you will also have to make that change in the ui-component PR. The same is true for the `error` property.

Reply:
Changes made to the UI to allow ReactNode. And the documentation updated to display `string | slot` for angular and `string | ReactNode` for React...for both helptext and error properties...

See the updated screenshot in the beginning.